api/src/main/java/jakarta/servlet/http/Cookie.java

```diff
@@ -83,22 +94,23 @@ public Boolean run() {
     //
     // The value of the cookie itself.
     //
-
-    private String name; // NAME= ... "$Name" style is reserved
+    private final String name; // NAME= ... "$Name" style is reserved
     private String value; // value of NAME
```

Comment:
Why is only name changed final but value not?

Reply:
Because there is a setter for the value, but not for the name. Thus only the name is immutable.